CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ v1.8.3 [2020-09-30]
 
 ### Bugfixes
 
+-   [#20101](https://github.com/influxdata/influxdb/pull/20101): fix(write): Successful writes increment write error statistics incorrectly
```

Comment:
This should move up to the `v1.8.4 [unreleased]` section

Reply:
Will do.

Reply:
Fixed